README.rst

```diff
@@ -173,6 +173,110 @@ To delete your local images and clear your local library::
+  ``composer``, ``date``, ``genre``, ``performer``, and ``max-age``.
+
+- ``local:directory?type=date``: References to directories grouping tracks by
+  date and album. Dates are transformed according to the optional parameter
+  ``FORMAT`` which default to ``%Y-%m-%d``. The URIs of the references start
+  with ``local:directory?date=``.
+
+- ``local:directory?type=genre``: References to directories named after genres
+  found among all tracks. Their URIs start with ``local::directory?genre=``.
+
+- ``local:directory?type=album``: References to all albums.
+
+- ``local:directory?type=album&PARAM=VALUE``: References to directories
+  grouping tracks matching the given criteria by album.  ``PARAM`` must be one
+  of ``albumartist``, ``artist``, ``composer``, ``date``, ``genre``,
+  ``performer``, ``max-age``. Their URIs start with
+  ``local::directory?PARAM=VALUE&type=track&album=local:album:``.
+
+- ``local:directory?type=artist``: References to all artists.
+
+- ``local:directory?type=artist&role=ROLE``: References to directories with URIs
+  ``local:directory?ROLE=URI`` where ``URI`` varies among all URIs starting with
+  ``local:artist`` build from all tracks tag corresponding to ``ROLE``. ``ROLE``
+  is one of ``albumartist``, ``artist``, ``composer``, or ``performer``.
+
+- ``local:directory?album=URI``: A reference to a directory grouping the tracks
+  of the album with given URI. Its URI starts with
+  ``local:directory?album=URI&type=track``.
+
+- ``local:directory?albumartist=URI``: References to directories grouping tracks
+  whose albumartist tag has given URI by album. Their URIs start with
+  ``local:directory?albumartist=URI&type=track&album=local:album:``.
+
+- ``local:directory?artist=URI``: References to directories grouping tracks
+  whose artist has given URI by album. Their URIs start with
+  ``local:directory?artist=URI&type=track&album=local:album:``.
+
+- ``local:directory?composer=URI``: References to directories grouping tracks
+  whose composer has given URI by album. Their URIs start with
+  ``local:directory?composer=URI&type=track&album=local:album:``.
+
+- ``local:directory?date=DATE``: References to directories grouping tracks whose
+  date match DATE by album. Their URIs start with
+  ``local:directory?date=DATE&type=track&album=local:album:``. The match is to
+  be interpreted as in a ``LIKE`` SQL statement.
+
+- ``local:directory?genre=GENRE``: References to directories grouping tracks
+  whose genre is GENRE by album. Their URIs start with
+  ``local:directory?genre=GENRE&type=track&album=local:album:``.
+
+- ``local:directory?performer=URI``: References to directories grouping tracks
+  whose performer has given URI by album. Their URIs start with
+  ``local:directory?performer=URI&type=track&album=local:album:``.
+
+- ``local:directory?max-age=SECONDS``: References to directories grouping tracks
+  whose date is newer than SECONDS seconds by album. Their URIs start with
```

Comment:
I think it's actually be the date it was scanned, not the date of the track itself.

Reply:
You're right.